src/attestation/azext_attestation/generated/_help.py

```diff
@@ -58,3 +58,13 @@
         text: |-
                az attestation delete --name "myattestationprovider" --resource-group "MyResourceGroup"
 """
+
+helps['attestation policy'] = """
+    type: group
+    short-summary: Manage the policies
+"""
+
+helps['attestation signer'] = """
+    type: group
+    short-summary: Manage the trusted policy signers
+"""
```

Comment:
Can you add examples here to convenient users?

Reply:
> Can you add examples here to convenient users?

Added.